chorus_book/src/guide-locations.md

```diff
@@ -29,3 +29,22 @@ The `ChoreographyLocation` trait provides the `name` method, which returns the n
 let name = Alice::name();
 assert_eq!(name, "Alice");
 ```
+
+## Location Set
+
+Each Choreography is allowed to operate on a set of `ChoreographyLocation`, called its `LocationSet`. You can use the macro `LocationSet!` and give it a comma separated list of `ChoreographyLocation` to build a `LocationSet`.
```

Comment:
I wonder if we need to mention "choreography" here as the readers don't know what it is at this point. Can we just explain what a location set is without talking about choreography or projector?